Description: Never default to using the custom title bar. Chromium should
  integrate, not be a special little snowflake.

Index: dev/ui/base/x/x11_util.cc
===================================================================
--- dev.orig/ui/base/x/x11_util.cc
+++ dev/ui/base/x/x11_util.cc
@@ -715,6 +715,7 @@ bool GetCustomFramePrefDefault() {
   // Also disable custom frames for (at-least-partially-)EWMH-supporting tiling
   // window managers.
   ui::WindowManagerName wm = GuessWindowManager();
+  return false;
   if (wm == WM_AWESOME || wm == WM_I3 || wm == WM_ION3 || wm == WM_MATCHBOX ||
       wm == WM_NOTION || wm == WM_QTILE || wm == WM_RATPOISON ||
       wm == WM_STUMPWM || wm == WM_WMII)
